tests/python/relax/test_backend_dispatch_sort_scan.py

```diff
@@ -399,5 +400,32 @@ def foo(x: R.Tensor((2, 3), "float32", "vulkan")):
     assert_structural_equal(mod, expected_mod)
 
 
+@tvm.testing.requires_cuda
+def test_dispatch_cumsum_gpu():
+    """Test cumsum kernel dispatch and numerical correctness"""
+
+    @I.ir_module
+    class Module:
+        @R.function
+        def main(x: R.Tensor(("m", "n"), "int32")):
+            with R.dataflow():
+                gv = R.cumsum(x, axis=-1, exclusive=False)
+                R.output(gv)
+            return gv
+
+    size = (8, 2000)
+    np_data = np.random.randint(0, 10, size).astype("int32")
+    np_cumsum = np.cumsum(np_data, axis=-1)
+    for target in ["cuda", "vulkan -supports_int64=1"]:
```

Comment:
Nitpick: Use `@tvm.testing.parametrize_targets("cuda", "vulkan -supports_int64=1")` instead of looping over each target.  This performs each test case in a separate pytest environment, 

* Exercises each test in a separate pytest case.  Can distinguish between failure on one specific backend as opposed to failure on every backend.
* Applies the appropriate `@tvm.testing.requires_*` marks for each target.  Currently, this test would fail if a developer runs it with `set(USE_CUDA ON)` and `set(USE_VULKAN OFF)`.

```python
@tvm.testing.parametrize_targets("cuda", "vulkan -supports_int64=1")
def test_dispatch_cumsum_gpu(target, dev):
   ...
```

Reply:
fixed in https://github.com/apache/tvm/pull/16947